round4/liuxk.md

```diff
@@ -0,0 +1,50 @@
+# Chrome Extension Idea: Web Styler
+
+## Authors
+
+Group 12: Kevin Liu, Adam Gorka, Andy Li, Judah Nouriyelian
+
+## Problem Statement
+
+By default, in order for users to change visual elements of webpages, they must manually inspect the underlying HTML/CSS, locate the desired elements, and change their states. This process is often time-consuming, if not difficult or even impossible for inexperienced or visually impaired users. We propose this extension as a way for users to customize these elements more easily.
+
+## Target Audience
+
+Targeted towards visually impaired individuals, specifically for colorblind and near-blind. Our extension will adjust the style of a website so that any text is larger. In addition, any coloring/style will be adjusted so that it's more clearly visible for individuals with color blindness.
+Many individuals that are color blind or visually impaired struggle with viewing web pages that are not formatted well. They would significantly benefit from a tool that enlarges text and changes colors to be color-blind friendly. 
+Additionally, the extension can also be used by other people who simply want an easy way to change such features for their own personal preferences.
+
+## Description
+
+This extension allows users to change visual features (e.g., text size, color) on websites to suit their preferences. Giving users a way to interact with these visual elements more directly and easily, the tool can be used for a variety of purposes, from accessibility to personalization.
```

Comment:
This is definitely an interesting and feasible extension to make. There are a few UX considerations to keep in mind to help non-technical users intuitively understand what and how they are changing certain features of a website.

**Some things to consider:**

1) The browser "Arc" has a similar capability called "Boosts" that you all should play around with and read about. They have thought a lot about stripping away the technical jargon and allowing users to playfully manipulate a website visually — https://www.theverge.com/2023/5/25/23735693/arc-browser-boosts-website-appearance-colors-features

2) The most important UX consideration is facilitating "smooth" and "continuous feedback" with the controls and the changes in the website. This means there should be a clear and immediate connection between the customisation settings you give the user and the content changing on the website, and ideally, these changes should occur smoothly for the best user experience.

For this, try minimising any delays and use "ease-out" or "spring" animation curves for interface changes with CSS or a library like "framer-motion". Such animations will lead to the _perception of the speed_ of your extension and the overall feel to be better, which is equally important as the _actual speed_.

3) See if you can think about giving people "just enough" customisation capabilities. Instead of allowing users to type in any font size in an input field, perhaps you can make it more intuitive and less daunting by defining presets of sizes to choose from. The same might apply to colours — writing in a HEX code is much more complicated than using a colour picker or choosing from a predefined palette. 

4) Some other interesting text properties to consider to increase accessibility could be line height, font type (usually serif fonts are easier to read), and perhaps even try to summarise content on a page to reduce reading (might be a stretch goal)